source/guides/language_support.html.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Language Support
+number: 6
+---
+# Language Support
+
+Lunr has full support for the indexing and searching of documents in English. If your documents are in another language, you need to install the [Lunr Languages](https://github.com/MihaiValentin/lunr-languages) plugin to get the best search results. Currently the following languages are supported:
+
+* <span class="flag flag-de">German</span>
+* <span class="flag flag-dk">Danish</span>
+* <span class="flag flag-es">Spanish</span>
+* <span class="flag flag-fi">Finnish</span>
+* <span class="flag flag-fr">French</span>
+* <span class="flag flag-hu">Hungarian</span>
+* <span class="flag flag-it">Italian</span>
+* <span class="flag flag-jp">Japanese</span>
+* <span class="flag flag-nl">Dutch</span>
+* <span class="flag flag-no">Norwegian</span>
+* <span class="flag flag-pt">Portuguese</span>
+* <span class="flag flag-ro">Romanian</span>
+* <span class="flag flag-ru">Russian</span>
+* <span class="flag flag-th">Thai</span>
+
+## Installing
+
+First, install the `lunr-languages` package:
+
+```shell
+$ npm install -g lunr-languages
+```
+
+Next, load the `lunr.stemmer.support` plugin and the appropriate language extension, which is identified by the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language code.
+
+The following example sets up the French language plugin and uses it when defining an index:
+
+```javascript
+var lunr = require("lunr")
+require("lunr-languages/lunr.stemmer.support")(lunr)
+require("lunr-languages/lunr.fr")(lunr)
+
+var idx = lunr(function () {
+  this.use(lunr.de)
+  this.ref('id')
+  this.field('text')
+
+  this.add({
+    id: 1,
+    text: "Ceci n'est pas une pipe"
+  })
+})
+```
```

Comment:
Could you add a simple example on how to use `idx` below? I think people not yet familiar with lunr would do the research, read this page and wonder "ok, I have an index. How do I search it now?".